pkg/cosesign1/ec384-cert.crt

```diff
@@ -0,0 +1,14 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
can the *.crt, *.body, *.pem, etc., files be moved to a `testda` directory?

Reply:
ok